reana_client/cli/workflow.py

```diff
@@ -16,10 +16,6 @@
 
 import click
 from jsonschema.exceptions import ValidationError
-from reana_commons.config import INTERACTIVE_SESSION_TYPES, REANA_COMPUTE_BACKENDS
-from reana_commons.errors import REANAValidationError
-from reana_commons.operational_options import validate_operational_options
-from reana_commons.utils import click_table_printer
```

Comment:
We normally keep this import order separated with an empty space:
- Python built-in modules
- External dependency modules
- Local imports

So the order here was correct.

Reply:
I see. Will revert it. My IDE is sorting it so need to be careful next time.

Reply:
Maybe this can be included in the [Contributing wiki section](https://github.com/reanahub/reana/wiki/Contributing) 😉

Reply:
Done